scalafmt-tests/src/test/scala/org/scalafmt/cli/CliTest.scala

```diff
@@ -457,15 +454,15 @@ trait CliTestBehavior { this: AbstractCliTest =>
       val conf = Cli.getConfig(args, opts)
       Cli.run(conf.get)
 
-      assertNoDiff(root / "scalatex.scalatex", unformatted)
-      assertNoDiff(root / "sbt.sbtfile", sbtOriginal)
+      assertNoDiff(dir2string(root / "scalatex.scalatex"), unformatted)
```

Comment:
what does `dir2string` do?

this file has a few similar changes (like `size` to `length`). are they related to munit change or just useful cleanup?

Reply:
> what does dir2string do?

[Reads](https://github.com/poslegm/scalafmt/blob/5ac6a99f179e79b5f7be80ad9b84ef91dc5b6e88/scalafmt-tests/src/test/scala/org/scalafmt/cli/FileTestOps.scala#L36) files from directory. (One file in this case)

Old `assertNoDiff` implementation from scalafmt did it itself, but MUnit doesn't know this custom logic.

All of changes except size -> length is MUnit related.